libs/block-editor/src/lib/extensions/dragHandler.extension.ts

```diff
@@ -132,7 +132,8 @@ export const DragHandler = (viewContainerRef: ViewContainerRef) => {
                                     nodeToBeDragged = getDirectChild(view.nodeDOM(position.inside));
                                     if (
                                         nodeToBeDragged &&
-                                        !nodeToBeDragged.classList?.contains('ProseMirror')
+                                        !nodeToBeDragged.classList?.contains('ProseMirror') &&
+                                        !nodeToBeDragged.innerText.startsWith('/')
```

Comment:
Move this to a method with a clear name, this will be getting bigger and bigger.